ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/loopback-interfaces.j2

```diff
@@ -9,33 +9,30 @@
 | Interface | Description | VRF | IP Address |
 | --------- | ----------- | --- | ---------- |
 {%     for loopback_interface in loopback_interfaces | arista.avd.natural_sort %}
-| {{ loopback_interface }} | {{ loopback_interfaces[loopback_interface].description | arista.avd.default('-') }} |
-{%-        if loopback_interfaces[loopback_interface].vrf is defined %} {{ loopback_interfaces[loopback_interface].vrf }} {%         else %} default {%         endif -%} |
-{%-        if loopback_interfaces[loopback_interface].ip_address is defined %}
- {{ loopback_interfaces[loopback_interface].ip_address }}
-{%-            if loopback_interfaces[loopback_interface].ip_address_secondaries is defined and loopback_interfaces[loopback_interface].ip_address_secondaries is not none %}
- <br> {{ loopback_interfaces[loopback_interface].ip_address_secondaries | join(' secondary <br> ') }} secondary
-{%-            endif %}
-{%-        else %}
- -
-{%-        endif %}
- |
+{%         set vrf = loopback_interfaces[loopback_interface].vrf | arista.avd.default('default') %}
+{%         if loopback_interfaces[loopback_interface].ip_address is arista.avd.defined %}
+{%             set ip = loopback_interfaces[loopback_interface].ip_address %}
+{%             if loopback_interfaces[loopback_interface].ip_address_secondaries is arista.avd.defined %}
+{%                 set ip = loopback_interfaces[loopback_interface].ip_address ~ ' <br> ' ~ loopback_interfaces[loopback_interface].ip_address_secondaries | join(' secondary <br> ') ~ ' secondary' %}
+{%             endif %}
+{%         endif %}
+| {{ loopback_interface }} | {{ loopback_interfaces[loopback_interface].description | arista.avd.default('-') }} | {{ vrf }} | {{ ip | arista.avd.default('-') }} |
 {%     endfor %}
 
 #### IPv6
 
 | Interface | Description | VRF | IPv6 Address |
 | --------- | ----------- | --- | ------------ |
 {%     for loopback_interface in loopback_interfaces | arista.avd.natural_sort %}
-| {{ loopback_interface }} | {{ loopback_interfaces[loopback_interface].description | arista.avd.default('-') }} |
-{%-        if loopback_interfaces[loopback_interface].vrf is defined %} {{ loopback_interfaces[loopback_interface].vrf }} {%         else %} default {%         endif -%} |
-{%-        if loopback_interfaces[loopback_interface].ipv6_address is defined %} {{ loopback_interfaces[loopback_interface].ipv6_address }} {%         else %} - {%         endif -%} |
+{%         set ip = loopback_interfaces[loopback_interface].vrf | arista.avd.default('default') %}
+{%         set ipv6 = loopback_interfaces[loopback_interface].ipv6_address | arista.avd.default('-') %}
+| {{ loopback_interface }} | {{ loopback_interfaces[loopback_interface].description | arista.avd.default('-') }} | {{ ip }} | {{ ipv6 }} |
```

Comment:
it would make it more consistent if you also set the `description` in a variable above. Same for ipv4 table